README.md

```diff
@@ -149,6 +149,12 @@ A small team of analytics engineers at Fivetran develops these dbt packages. How
 
 We highly encourage and welcome contributions to this package. Check out [this dbt Discourse article](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) on the best workflow for contributing to a package!
 
+## Opinionated Modelling Decisions
+
+The ServiceNow Connector provides thousands of tables. We developed our data models based on the tables with highest usage across Fivetran customers. 
+
+For more information on table relationships and how they informed our model development, you may refer to the [DECISIONLOG](DECISIONLOG.md).
```

Comment:
Instead of highlighting the table usage I would recommend we do a larger callout of why the table relationships may be confusing and how we treated them in the package. Then you can link to the DECISIONLOG for more info. Just to give a bit more detail around why this was included.

Reply:
Updated-- let me know what you think @fivetran-joemarkiewicz

Reply:
I think this new wording is great, thanks!